docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -17,6 +17,81 @@
 
 Converts `value` to [Decimal](../../data_types/decimal.md) of precision `S`. The `value` can be a number or a string. The `S` (scale) parameter specifies the number of decimal places.
 
+## toDecimal*OrNull
+
+Converts an input string to the value of [Nullable(Decimal(P,S))](../../data_types/decimal.md) data type. This family of functions include:
+
+- `toDecimal32OrNull( expr, S)` — Results with Nullable(Decimal32(S)) data type.
+- `toDecimal64OrNull( expr, S)` — Results with Nullable(Decimal64(S)) data type.
+- `toDecimal128OrNull( expr, S)` — Results with Nullable(Decimal128(S)) data type.
+
+**Parameters**
+
+- `expr` — Any of ClickHouse [expressions](../syntax.md#syntax-expressions), returning a value of the [String](../../data_types/string.md) data type. For example, `'1.111'`.
```

Comment:
Here we expect text representation of decimal. But allow any other string instead. The main reason to have this function is to allow user to convert some sort of strings into decimals without errors that occurs if we use toDecimal* with wrong value. toDecimal64OrNull allow to parse such bad formated string columns without exceptions.